corehq/ex-submodules/casexml/apps/case/templatetags/case_tags.py

```diff
@@ -47,6 +47,7 @@ def render_case(case, options):
     from corehq.apps.hqwebapp.templatetags.proptable_tags import get_tables_as_rows, get_definition
     case = wrapped_case(case)
     timezone = options.get('timezone', pytz.utc)
+    timezone = timezone.localize(datetime.datetime.now()).tzinfo
     _get_tables_as_rows = partial(get_tables_as_rows, timezone=timezone)
```

Comment:
This error isn't everywhere - for example, the the form submission times on that same page referenced in [the ticket](http://manage.dimagi.com/default.asp?154276) uses IST, or at least claims to.

I think the way this **should** work is that the timezone should be used to localize dates before they're rendered, rather than trying to construct a new date with that timezone as an argument.  Daylight savings time in particular makes it tricky to localize a timezone once and call it a day (and I'm not actually sure where/how that'd work either).
Here's the problem:
```python
>>> import pytz, datetime
>>> submission_time = datetime.datetime.now()
>>> tz = pytz.timezone('Asia/Kolkata')
>>> 
>>> # bad way
>>> print submission_time.replace(tzinfo=tz)
2015-01-29 16:29:51.146194+05:53
>>> print submission_time.replace(tzinfo=tz).tzname()
HMT
>>> 
>>> # good way
>>> print tz.localize(submission_time)
2015-01-29 16:29:51.146194+05:30
>>> print tz.localize(submission_time).tzname()
IST
>>> 
```
(note that the bad way gives you an offset of `5:53` and the old abbreviation, instead of the desired `5:30` and `IST`)

Neither of these methods affect the actual time - it's not a conversion between time zones, just different ways of attaching timezone information to a `date`/`datetime` object without a specified timezone.